packages/app/src/runner/index.ts

```diff
@@ -427,12 +405,22 @@ async function executeSpec (spec: SpecFile, isRerun: boolean = false) {
 
   UnifiedReporterAPI.setupReporter()
 
+  // TODO: UNIFY-1318 - figure out how to manage window.config.
+  const config = getRunnerConfigFromWindow()
+
+  // this is how the Cypress driver knows which spec to run.
+  config.spec = setSpecForDriver(spec)
+
+  // creates a new instance of the Cypress driver for this spec,
+  // initializes a bunch of listeners watches spec file for changes.
+  getEventManager().setup(config)
+
```

Comment:
Thanks for reducing this duplication.

Reply:
No probs! Most of this could be reduced actually but kept it simple to stay in scope